docs/user-guide/apis/python-api.md

```diff
@@ -0,0 +1,75 @@
+# Using the Python API
+
+This section covers an overview of the python API extension for Gaffer to
+demonstrate how to get up and running to perform queries from Python code on an
+existing running graph.
+
+!!! tip
+    Please see the handy introduction on [what is python](../gaffer-basics/what-is-python.md)
+    if you are new to the language.
+
+## What is the Python Extension?
+
+Commonly referred to as `gafferpy` this is an API to gaffer that provides
+similar querying capabilities to the rest API but from Python. Fundamentally it
+wraps the rest API to use the same JSON under the hood this means you should be
+able to access almost any features or end points available in the main rest API.
+
+## Installation
+
+Currently there isn't a release of `gafferpy` on pypi or other pip repository;
+however, the source code can still be cloned from the [git repository](https://github.com/gchq/gaffer-tools/tree/develop/python-shell)
+and installed via pip. Please see the readme in the `gafferpy` repository for
```

Comment:
link the readme?

Reply:
I could do, I kind of didn't put it in here as there was a link to the repository in the sentence directly before which will render the readme when you access it